main.yml action errors out on `docker rmi $(docker image ls -aq)` command

The main.yml action file error out on the command
		
`docker rmi $(docker image ls -aq)`
		

with the error:

```bash
docker: 'docker rmi' requires at least 1 argument

Usage:  docker rmi [OPTIONS] IMAGE [IMAGE...]

See 'docker rmi --help' for more information
Error: Process completed with exit code 1.
```

This is because the `$(docker image ls -aq)`  sub command is returning nothing. (because there are no previous images)
This is likely because the command has not been running for months.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -15,7 +15,7 @@ jobs:
         sudo swapoff -a
         sudo rm -f /swapfile
         sudo apt clean
-        docker rmi $(docker image ls -aq)
+        docker image prune -a -f
         df -h
     - uses: actions/checkout@v3
     - name: Set up Python

